src/classes/ffhttp_AbstractFormDataSerializable.cls-meta.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ApexClass xmlns="http://soap.sforce.com/2006/04/metadata">
+    <apiVersion>38.0</apiVersion>
```

Comment:
Think we should look at updating the apiVersion for all meta data files

Reply:
Is that in the scope of this change?

Reply:
Leaving for now, as discussed.